lib/crawly/settings.ex

```diff
@@ -43,6 +43,7 @@ defmodule Crawly.Settings do
           # Defines retries
           retry: retry(),
           middlewares: [middleware()],
-          pipelines: [pipeline()]
+          pipelines: [pipeline()],
+          on_spider_closed_callback: function()
```

Comment:
Do you think it'll be good to nest it under an `events` or `callbacks` key? For organisational purpose

Reply:
I am not quite sure, from one side I don't want to have nested sections, as the configuration will be more complex. Lets maybe keep it as it is for now, and see if we need to tweak later.